packages/leavitt-elements/src/leavitt-person-company-select.ts

```diff
@@ -126,14 +134,23 @@ export class LeavittPersonCompanySelectElement extends LoadWhile(LitElement) {
     this.searchTerm = '';
   }
 
+  /**
+   *  Sets focus on the input.
+   */
   focus() {
     this.textfield.focus();
   }
 
+  /**
+   *  Returns true if the input passes validity checks.
+   */
```

Comment:
Looks like we need an ignore when it does
      this.dispatchEvent(new LeavittPersonCompanySelectSelectedEvent(this.selected));
as well, otherwise it is still trying to generate a blank event in the docs